storybook/cypress/e2e/percy-voorbeeldpaginas-html-css.cy.ts

```diff
@@ -0,0 +1,19 @@
+import voorbeeldpaginasHtmlCss from "../fixtures/percy-voorbeeldpaginas-html-css.json";
+
+describe("Percy", () => {
+  if (!Cypress.env("PERCY")) {
+    it("should not take screenshots", () => {
+      cy.wrap(true).should("eq", true);
+    });
+
+    return;
+  }
+
+  it("take screenshot of all in /cypress/fixtures/percy-voorbeeldpaginas-html-css.json", () => {
+    for (const id of voorbeeldpaginasHtmlCss) {
+      cy.visit(`http://localhost:45000/iframe.html?id=${id}`);
+
+      cy.wait(500).percySnapshot(id);
```

Comment:
@LukeVerlaan Dit is een test om te kijken of het een async probleem is of dat we verder moeten kijken.